jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -603,6 +610,43 @@ void java_bytecode_parsert::get_class_refs()
   }
 }
 
+/// For each of the given annotations, get a reference to its class and
+/// recursively get class references of the values it stores.
+void java_bytecode_parsert::get_annotation_refs(
```

Comment:
Maybe `get_annotation_class_refs` as it's not references to the annotation that you're getting.